cdap-watchdog/src/main/java/co/cask/cdap/logging/appender/LogAppender.java

```diff
@@ -37,6 +38,8 @@ public final void append(ILoggingEvent eventObject) {
         return;
       }
     }
+    // Check if this is a User Log and add appropriate tag
+    CondensedLogFilter.addUserLogTag(eventObject);
```

Comment:
LogAppender shouldn't be calling filter. Filter should only be used in query (log fetching) time.